tests/smoke/fake_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEoAIBAAKCAQEAz0BXcFS6/mpbFiIGEc/Tnag4Z8fY47hW5HUnaaquIHfbmQKW
+JYrzAw+FIIxWs0KXs8fA4jYbvcpu4ui4AHHcIvtdIROC/cqCb/bgYGwcTgHeuc56
+i6hfdTbVeJpSj+WHybUexIbPqUwfiCAvqOb3g2z5/gWIAzCdsezrG59f4BUg5Hp/
+8qDhW2G0TBjH1REGIycwuK/6gih9G5f8blTS8X/RZFkkwnEBShjp3cLhiEw5LIQ+
+K9ipPIFHXVz8aGoQb99/9kB9AmOc2ymI4YDeiOlrcDvjtbicLTXldQBoWI7q3Xet
+KoIDcMflu+tt7it5yF7yXd4cwzvLuaGud/7SCwIDAQABAoIBAAWyBfUfTvg44gHs
+s2//3xfhDJPFiS9Q3IpwscGUoC5iPNWqnYewltcJsSXg1W0dsl+NaL6OsBp6Gqv/
+qzbP6jXH5JpJF/tzejwrc6USs4BtSrqecv2thdoJxLoIu6Yar62aXZsW/VPwtJkg
+EU858alZ8FDLxB4aUusV0Kw0qATXYxYdSldVN7nnG1OoUZKzbE32ZgWe6ZvoRUYF
+EyIYfExnw2BolhIRcO3M7f9FwXNWcZ30UyVgXlkfbD9mvfvKB+x+cWa5vn1hi0x0
+KQAMaYnar3IZB11OMSOh4t3JNQ0/BQn10vPUYtcayHKFvmIo8h4PqjklIneMZ0OF
+kUxPLGECgYEA/LolE4WOzA//wG5Ju1plgC4Xhd2+3eRrUlPwfIVhT94IzbsIQ6Ba
+bt2MBsqTeHI8cblewR7keSQXxpeQB+tfkiMIsBsgOAMXaTIVTzkF5zyiv3WVyvPd
+ZGUy+Lq4h+0Y6i+FHCO5QXUKOUX66ouV+TDLNIwPIRrNhcg01ss9pHECgYEA0e9u
+zWvao5APwtxLrPTFiOMxf+zcd+XSmuO/yM4j7UXGXHn8zf4yAYuzJv2dU5v5wNrG
+8u4JH0Cr1M3neyorXocQ0j+ZxXoayA7uNV06WXsmygCNJ+tIdn3PHRpb8SDTWan8
+q8x1FRFCEp5H8dgW9JR7HDrNMLjwE6KPREourDsCgYB5SWxZvhMV4+e9ljBUdcGY
+joZDfHQJUYIf9KDJYcmd5tUaxtPkgKRl6HU3HSlA+LIBchEDZawIZ0XxzPMWmeM3
+4xBcU+hWeUVW+u43IwquOu9wKyJEUFzDvb2EmTfOI/99BGlBY/OYcm8BpZAR2AOV
+n9El2sG4BYD032xcWRWwQQKBgHQspsmA1PZMo5w24p3sul+dMmhTgFHNWbWjMR+D
+i/gi87l7PD4WXD/kuhhqoqSBHT3vE8edmGi2FiDmbSIIXQWrgtCGp6bTgnyRXpsj
+Y0XfU8DXe0XK+0evKW02iAixHKRy4EOGGoi1CzZYZwHbjiB9Bn/Bvokm3pxPCi9F
+Sfy/An99QZuZ1Q0079po7z9B/LdHz31KpSoT1X8/KFzA5dHjzPRUG+9qYSTdaOIZ
+KtcTQ5A+9weZ0495YA1w1RIli8Siy//MQAU+8t9RoizJ8omjFCGuOBp1BLTRcT1D
+Rpkx1qk6n2Yt+sVDUHEgEPVpstwmzO+JXl+V0mcFnCiBJWq3
+-----END RSA PRIVATE KEY-----
```

Comment:
We'd need to have to have a templating system for the matchbox groups (which is itself a variable file for templating Ignition).